Assets/Mirror/Components/NetworkRoomManager.cs

```diff
@@ -194,7 +194,7 @@ public void CheckReadyToBegin()
             OnRoomServerPlayersReady();
         }
 
-        void CallOnClientEnterRoom()
+        public void CallOnClientEnterRoom()
```

Comment:
Why does this need to be public?

Reply:
So it can be called from NetworkRoomPlayer.cs, it could be internal instead

Reply:
Will look into this one more this evening.